config/database.yml

```diff
@@ -56,9 +56,9 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: CareerAtlas_test
-
+  adapter: postgresql
+  database: travis_ci_test
```

Comment:
Doesn't this break your tests locally? you changed your db name?   Can't we just ask travis to make a database with the same name?     Inside of our `.travis.yml` we need to run this `psql -c 'create database CareerAtlas_test;' -U postgres` as a before script.   That will ensure that the db exists on travis prior to tests running.